src/api-reference/04.data-types.md

```diff
@@ -155,7 +155,7 @@ separators (in cents).
 
 Shipment status.
 
-<DataTypeDefinition pattern="/\d+/" example="1 pending - concept,2 pending - registered,3 enroute - handed to carrier,4 enroute - sorting,5 enroute - distribution,6 enroute - customs,7 delivered - at recipient,8 delivered - ready for pickup,9 delivered - package picked up,10 delivered - return shipment ready for pickup,11 delivered - return shipment package picked up,12 printed - letter,13 inactive - credited,14 printed - digital stamp,18 printed - untracked shipment,30 inactive - concept,31 inactive - registered,32 inactive - enroute - handed to carrier,33 inactive - enroute - sorting,34 inactive - enroute - distribution,35 inactive - enroute - customs,36 inactive - delivered - at recipient,37 inactive - delivered - ready for pickup,38 inactive - delivered - package picked up" />
+<DataTypeDefinition pattern="/\d+/" example="1 pending - concept,2 pending - registered,3 enroute - handed to carrier,4 enroute - sorting,5 enroute - distribution,6 enroute - customs,7 delivered - at recipient,8 delivered - ready for pickup,9 delivered - package picked up,10 delivered - return shipment ready for pickup,11 delivered - return shipment package picked up,12 printed - letter,13 inactive - credited,14 printed - digital stamp,15 printed - external shipment,16 expired,17 cancelled,18 printed - untracked shipment,19 delivered - at agreed location,30 inactive - concept,31 inactive - registered,32 inactive - enroute - handed to carrier,33 inactive - enroute - sorting,34 inactive - enroute - distribution,35 inactive - enroute - customs,36 inactive - delivered - at recipient,37 inactive - delivered - ready for pickup,38 inactive - delivered - package picked up" />
```

Comment:
Hierboven is `13` gecorrigeerd naar `credited`, dat moet hier dan ook, neem ik aan.

Reply:
Je hebt gelijk, scherp! ✅